README.md

```diff
@@ -14,31 +14,42 @@ and [examples](https://github.com/redhat-developer/opencompose/blob/master/examp
 We are open to suggestions and contributions from the Kubernetes community as our project grows.
 Please send any PRs, issues or RFCs to improve this project.
 
-### Installation
-#### Binaries
+## Installation
+
+#### Binary installation
+
 You can retrieve binaries for Linux, macOS and Windows on our [GitHub release page](https://github.com/redhat-developer/opencompose/releases).
 
-##### Linux
-If you prefer to do it straight from CLI here's a one-liner for you:
-```bash
+``sh
+# Linux 
```

Comment:
@cdrage These don't render correctly. The # Linux breaks it, at least in the preview mode. Same applies below as well.